CMakeLists.txt

```diff
@@ -900,6 +900,11 @@ if(NOT DEFINED BUILD_SHARED_LIBS)
 else()
     add_library(zlib ${ZLIB_ALL_SRCS})
 
+    if(NOT BUILD_SHARED_LIBS)
+		# Add zlibstatic alias
```

Comment:
Please replace tabs with spaces and make sure indentation is correct.

Reply:
I don't think the comment is necessary.

Reply:
Force-pushed the changes. 👍